hmcts/pre-api
# S28 2972: Check Stream Endpoint

### JIRA ticket(s)

- S28-2972


### Change description
- Add endpoint /media-service/live-event/check/{captureSessionId} to check stream





> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**
- Test: Capture session id that doesn't exist- see 404
- Test: Capture session id for capture session that has been marked as deleted- see 404
- Test: Capture session with status = RECORDING- see 200 (capture session returned)
- Test: Capture session with finished at not null- see 400 (capture session already finished)
- Test: Capture session with started at = null- see 400 (capture session not started)
- Test: Capture session with status != STANDBY- see 400 (wrong status)
- Test: Create a capture session and then start live event- call endpoint. See 200 (see capture session status is still STANDBY- therefore stream not available).
- Test: Create a capture session. Start live event. Start the CVP recording. Call endpoint- see 200 (capture session status is RECORDING- therefore stream available).
